ecommerce/enterprise_coupons/utils.py

```diff
@@ -0,0 +1,38 @@
+"""
+Helper methods for enterprise app.
+"""
+import logging
+
+from django.conf import settings
+from .backends import get_coupon_backend
+
+log = logging.getLogger(__name__)
+
+def get_enterprise_catalog_id_from_coupon(coupon_code, backend_alias='default'):
```

Comment:
I guess this is not a util function, this should be part of the backends module, what do you think? @daviidco